app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="mt-10 sm:mt-0 max-w-screen-lg px-4 py-8 mx-auto">
+  <div class="md:grid md:grid-cols-3 md:gap-6">
+    <div class="mt-5 md:col-span-2 md:mt-0">
+    <h1 class="pl-5 text-lg">Create a New Group</h1>
+      <%= form_with(model: group) do |form| %>
+        <div class="overflow-hidden shadow sm:rounded-md">
+          <div class="bg-white px-4 py-5 sm:p-6">
+          <% if @group.errors.any? %>
+            <% @group.errors.full_messages.each do |message|%>
+                <h3><%= message %> </h3>
+            <%end%>
+        <%end%>
+            <div class="grid grid-cols-6 gap-6">
+              <div class="col-span-6 sm:col-span-3">
+                <label for="first-name" class="block text-sm font-medium text-gray-700">name</label>
+                <%= form.text_field :name, class: "mt-1 block w-full border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm" %>
+              </div>
+
+              <div class="col-span-6 sm:col-span-3">
+                <label for="last-name" class="block text-sm font-medium text-gray-700">Icon Link</label>
+                <%= form.text_field :icon, class: "mt-1 block w-full border-gray-300 shadow-sm focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm" %>
+              </div>
```

Comment:
- [ ] I could create a group without filling out this field. Let's kindly make sure to validate all user inputs.